app/controllers/user_controller.rb

```diff
@@ -6,4 +6,14 @@ def show
     @wiki = Wiki.new
     @wikis = @user.wikis
   end
+
+  def downgrade
+    current_user.update_attributes!(role: 'standard')
+
+    if current_user.role == 'standard'
+      flash[:notice] = "Membership Downgraded!"
+    else
+      flash[:error] = "There was an error downgrading your membership. Please try again."
+    end
+  end
```

Comment:
@bjblock - I put the `downgrade` action within the user_controller as we're updating user attributes, is this correct? I realized while doing this, that the options to upgrade/downgrade should be in the `user_show` page rather than a `charges_new` page which I'll implement once I get this to work properly.